.travis.yml

```diff
@@ -13,7 +13,5 @@ matrix:
       env: TOXENV=py36
     - python: 3.7
       env: TOXENV=py37
-    - python: 3.6
-      env: TOXENV=flake8
```

Comment:
I really don't think we should drop python3.6, since that's what RHEL 8 has.

Reply:
Are you suggesting to drop py37 flake8 run instead?

Reply:
Currently we're doing both in TravisCI, and when you run `tox` on a system that has one python3 version and not the other, it just runs the one you have.

Reply:
Ah, ok. I considered this double flake8 run in TravisCI a bit confusing duplication, but if you are worried that it could stop working with either one of py37 or py36, I'm going to bring it back.